spyderlib/plugins/editor.py

```diff
@@ -944,7 +952,7 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 

```

Comment:
Not sure if we want to have this option added to the file menu. I would say no it should not

@ccordoba12, @blink1073?